tests/ExampleApps/HeadlessLibAndApp/libHeadless/CMakeLists.txt

```diff
@@ -3,7 +3,19 @@ if(APPLE)
     cmake_policy(SET CMP0042 NEW)
 endif(APPLE)
 
-project(RadiumHeadless VERSION 1.0.0)
+project(RadiumHeadless VERSION 1.0.2)
+
+# Defines the RadiumHeadless version in the parent scope so that on can use this lib doing
+# add_subdirectory(...) and access to the
```

Comment:
```suggestion
# Defines the RadiumHeadless version in the parent scope so that one can use this lib doing
# add_subdirectory(...) and access RadiumHeadless_VERSION
```